src/decode.c

```diff
@@ -3027,15 +3026,15 @@ get_base_value_type(short gc)
               if (gc <= 409) return VT_INT16;
               if (gc <= 419) return VT_STRING;
               if (gc <= 429) return VT_INT32;
-              if (gc <= 439) return VT_STRING;
+              /*if (gc <= 439)*/ return VT_STRING;
```

Comment:
why? doesn't look right

Reply:
This not a mistake, just unnecessary `if`. This comparison is always true

Reply:
I see. But I rather want to keep it in nevertheless then, please.